docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -126,6 +124,10 @@ The Player sub-APIs offer the following features:
 
   After changing to the READY state, the subtitles can be read.
 
+- Media Streams
+
+  Use the media stream API(in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__SUBTITLE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__SUBTITLE__MODULE.html) applications) to play and control the external demuxed stream.
```

Comment:
a space can be given between API and (in mobile

Reply:
I will update at next patch. Thank you.

Reply:
it is updated.